crates/sncast/src/lib.rs

```diff
@@ -63,6 +64,24 @@ impl FromStr for AccountType {
     }
 }
 
+impl AccountType {
+    fn to_string_pretty(self) -> String {
+        let repr = match self {
+            AccountType::Oz => "OpenZeppelin",
+            AccountType::Argent => "Argent",
+            AccountType::Braavos => "Braavos",
+        };
+
+        repr.to_string()
```

Comment:
Function `to_string_pretty` operates on the `AccountType` enum itself. In fact, it exists only because of the `AccountType::Oz` variant which  isn't descriptive enough to print. In my opinion, it should be renamed to `AccountType::OpenZeppelin` for consistence with other variants.

Reply:
I am strictly talking about the fallback `.to_string()` here
Though if you want to rename the enum variants, go ahead